### PR TITLE
[Security Rules] Update security rules package to v1.0.2

### PR DESCRIPTION
**Pull request by terrancedejesus**
## What does this PR do?
Update the Security Rules package to version 1.0.2.
Autogenerated from commit  https://github.com/elastic/detection-rules/tree/5fced6368bcc44431039ae80929d42d47e8cdd69

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] ~I have verified that all data streams collect metrics or logs.~
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist
- Install the most recently release security rules in the Detection Engine
- Install the package
- Confirm the update is available in Kibana. Click "Update X rules" or "Install X rules"
- Look at the changes made after the install and confirm they are consistent

## How to test this PR locally
- Perform the above checklist, and use `package-storage` to build EPR from source

## Related issues
None

## Screenshots
None

### CODE REVIEW

File: packages/security_detection_engine/docs/README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ To download or update the rules, click **Settings** > **Install Prebuilt Securit
 Then [import](https://www.elastic.co/guide/en/security/master/rules-ui-management.html#load-prebuilt-rules)
 the rules into the Detection engine.
 
+## License Notice
+
```
Comment (brokensound77) [inline packages/security_detection_engine/docs/README.md:10]:
```suggestion
```

File: packages/security_detection_engine/kibana/security_rule/699e9fdb-b77c-4c01-995c-1c15019b9c43.json [hunk 2]
```diff
@@ -186,17 +186,17 @@
...
                 ]
             }
         ],
-        "threat_query": "@timestamp \u003e= \"now-30d\" and event.module:threatintel and (threatintel.indicator.file.hash.*:* or threatintel.indicator.file.pe.imphash:* or threatintel.indicator.ip:* or threatintel.indicator.registry.path:* or threatintel.indicator.url.full:*)",
+        "threat_query": "@timestamp \u003e= \"now-30d/d\" and event.module:threatintel and (threat.indicator.file.hash.*:* or threat.indicator.file.pe.imphash:* or threat.indicator.ip:* or threat.indicator.registry.path:* or threat.indicator.url.full:*)",
         "timeline_id": "495ad7a7-316e-4544-8a0f-9c098daee76e",
         "timeline_title": "Generic Threat Match Timeline",
         "type": "threat_match",
-        "version": 4
+        "version": 3
```
Comment (brokensound77) [inline packages/security_detection_engine/kibana/security_rule/699e9fdb-b77c-4c01-995c-1c15019b9c43.json:198]: 👀

File: packages/security_detection_engine/kibana/security_rule/785a404b-75aa-4ffd-8be5-3334a5a544dd.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (brokensound77) [inline packages/security_detection_engine/kibana/security_rule/785a404b-75aa-4ffd-8be5-3334a5a544dd.json:37]: 👀

File: packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json [hunk 4]
```diff
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json:54]: Version 10 in [Version Lock](https://github.com/elastic/detection-rules/blob/c803160e4f513f865986e66e323a10cc59be9d34/etc/version.lock.json) file.

Comment (brokensound77) [reply packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json:54]: 11 in prod [0.16.2](https://epr.elastic.co/package/security_detection_engine/0.16.2/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json)

File: packages/security_detection_engine/kibana/security_rule/e555105c-ba6d-481f-82bb-9b633e7b4827.json [hunk 5]
```diff
@@ -31,7 +31,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 12
+        "version": 11
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/e555105c-ba6d-481f-82bb-9b633e7b4827.json:34]:
```suggestion
        "version": 13
```

File: packages/security_detection_engine/kibana/security_rule/785a404b-75aa-4ffd-8be5-3334a5a544dd.json (hunk 3 above)
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/785a404b-75aa-4ffd-8be5-3334a5a544dd.json:37]:
```suggestion
        "version": 12
```
This is a result of a bug found in the version locks and forked rules as being discussed/worked in https://github.com/elastic/detection-rules/issues/1949

Bumping to 12.

File: packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json (hunk 4 above)
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/68994a6c-c7ba-4e82-b476-26a26877adf6.json:54]:
```suggestion
        "version": 12
```

Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949

File: packages/security_detection_engine/kibana/security_rule/ad3f2807-2b3e-47d7-b282-f84acbbe14be.json [hunk 6]
```diff
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/ad3f2807-2b3e-47d7-b282-f84acbbe14be.json:54]:
```suggestion
        "version": 12
```

Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949

File: packages/security_detection_engine/kibana/security_rule/acbc8bb9-2486-49a8-8779-45fb5f9a93ee.json [hunk 7]
```diff
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/acbc8bb9-2486-49a8-8779-45fb5f9a93ee.json:54]:
```suggestion
        "version": 12
```

File: packages/security_detection_engine/kibana/security_rule/a99f82f5-8e77-4f8b-b3ce-10c0f6afbc73.json [hunk 8]
```diff
@@ -31,7 +31,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 12
+        "version": 11
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/a99f82f5-8e77-4f8b-b3ce-10c0f6afbc73.json:34]:
```suggestion
        "version": 13
```
Bumping as a result of bug identified: https://github.com/elastic/detection-rules/issues/1949

File: packages/security_detection_engine/kibana/security_rule/cad4500a-abd7-4ef3-b5d3-95524de7cfe1.json [hunk 9]
```diff
@@ -34,7 +34,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 12
+        "version": 11
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/cad4500a-abd7-4ef3-b5d3-95524de7cfe1.json:37]:
```suggestion
        "version": 13
```

File: packages/security_detection_engine/kibana/security_rule/6f435062-b7fc-4af9-acea-5b1ead65c5a5.json [hunk 10]
```diff
@@ -51,7 +51,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/6f435062-b7fc-4af9-acea-5b1ead65c5a5.json:54]:
```suggestion
        "version": 12
```

File: packages/security_detection_engine/kibana/security_rule/cf549724-c577-4fd6-8f9b-d1b8ec519ec0.json [hunk 11]
```diff
@@ -34,7 +34,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (brokensound77) [inline packages/security_detection_engine/kibana/security_rule/cf549724-c577-4fd6-8f9b-d1b8ec519ec0.json:37]: 👀

Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/cf549724-c577-4fd6-8f9b-d1b8ec519ec0.json:37]:
```suggestion
        "version": 12
```

File: packages/security_detection_engine/kibana/security_rule/93e63c3e-4154-4fc6-9f86-b411e0987bbf.json [hunk 12]
```diff
@@ -34,7 +34,7 @@
         ],
         "timestamp_override": "event.ingested",
         "type": "query",
-        "version": 11
+        "version": 10
```
Comment (terrancedejesus) [inline packages/security_detection_engine/kibana/security_rule/93e63c3e-4154-4fc6-9f86-b411e0987bbf.json:37]:
```suggestion
        "version": 12
```

File: packages/security_detection_engine/kibana/security_rule/699e9fdb-b77c-4c01-995c-1c15019b9c43.json (hunk 2 above)
Comment (brokensound77) [reply packages/security_detection_engine/kibana/security_rule/699e9fdb-b77c-4c01-995c-1c15019b9c43.json:198]: looks like a github diff rendering issue - it is mis-associating a change

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**